### PR TITLE
feat(infra-apps): kubePrometheusStack 32.2

### PR DESCRIPTION
**Pull request by kebarb**
# Description

updates kubePrometheusStack to target version 32.2.*

- [upgrade instructions](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack#from-31-x-to-32-x)

# Changes

This upgrades to prometheus-operator to v0.54.0 and prometheus to v2.33.1. It also changes the default for grafana.serviceMonitor.enabled to `true`.

If you are using label base ServiceMonitor discovery you will need to expplicitly configure the label for grafana rather than reliying on just `commonLabels:

```yaml
grafana:
  serviceMonitor:
    enabled: true
    labels:
      k8s.example.com/label: value
```

- https://github.com/prometheus-community/helm-charts/pull/1804
- https://github.com/prometheus-community/helm-charts/pull/1802
- https://github.com/prometheus-community/helm-charts/pull/1798
- https://github.com/prometheus-community/helm-charts/pull/1788
- https://github.com/prometheus-community/helm-charts/pull/1789
- https://github.com/prometheus-community/helm-charts/pull/1583
- https://github.com/prometheus-community/helm-charts/pull/1782
- https://github.com/prometheus-community/helm-charts/pull/1757
- https://github.com/prometheus-community/helm-charts/pull/1783
- https://github.com/prometheus-community/helm-charts/pull/1687
- https://github.com/prometheus-community/helm-charts/pull/1777
- https://github.com/prometheus-community/helm-charts/pull/1778
- https://github.com/prometheus-community/helm-charts/pull/1772
- https://github.com/prometheus-community/helm-charts/pull/1732
- https://github.com/prometheus-community/helm-charts/pull/1770
- https://github.com/prometheus-community/helm-charts/pull/1730
- https://github.com/prometheus-community/helm-charts/pull/1766
- https://github.com/prometheus-community/helm-charts/pull/1767
- https://github.com/prometheus-community/helm-charts/pull/1759
- https://github.com/prometheus-community/helm-charts/pull/1763
 
# Issues

closes https://github.com/adfinis-sygroup/helm-charts/issues/559

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [ ] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/infra-apps/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
-version: 0.77.1
+version: 0.77.2
```
Comment (tongpu) [inline charts/infra-apps/Chart.yaml:5]: Our standard policy is to bump the minor version of our chart of we bump any version of a chart included in the update:
```suggestion
version: 0.78.0
```

**Review by tongpu — DISMISSED** [review]

File: charts/infra-apps/Chart.yaml (hunk 1 above)
Comment (kebarb) [reply charts/infra-apps/Chart.yaml]: pushed changes

**Comment by tongpu** [comment]
Could you please check the labels on the ServiceMonitor deployed from the grafana sub-chart. I have a feeling that we probably have to configure [`grafana.serviceMonitor.labels`](https://github.com/prometheus-community/helm-charts/blob/0fbb2d201698b65388a03a3f15ec0eb026b3973d/charts/kube-prometheus-stack/values.yaml#L784) to have this picked up by prometheus-operator.

**Review by hairmare — APPROVED** [review]
Thanks for looking into the label thing, i added a warning about it to the PR description and this now LGTM

**Comment by eyenx** [comment]
I am opening an internal ticket to track this. thanks @kebarb

**Review by eyenx — APPROVED** [review]

**Pull request merged**